### PR TITLE
Actively developing caffe parser. Do not merge

### PR DESCRIPTION
**Pull request by jyt109**

### CODE REVIEW

**Comment by JozoVilcek** [comment]
Will this pull reqeust cover #179? Do you have any ETA on this?

**Comment by agibsonccc** [comment]
This will likely be part of its own minor release. We're done with adapting the configuration and will be getting this up and running in the coming weeks. Right now loading the files in to java is done. If you want it you're welcome to push to this branch.

File: dl4j-caffe/src/main/java/org/deeplearning4j/caffe/common/NNCofigBuilderContainer.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import lombok.NoArgsConstructor;
+import org.deeplearning4j.nn.conf.NeuralNetConfiguration.Builder;
+import org.deeplearning4j.nn.conf.NeuralNetConfiguration.ListBuilder;
+
+/**
+ * @author jeffreytang
+ */
+@Data
+@NoArgsConstructor
+public class NNCofigBuilderContainer {
```
Comment (EronWright) [inline dl4j-caffe/src/main/java/org/deeplearning4j/caffe/common/NNCofigBuilderContainer.java]: "NNConfigBuilderContainer"

**Comment by EronWright** [comment]
Please re-open the PR when the code is ready for review, thanks.

**Comment by oak-tree** [comment]
what is the status of this issue? is it possible to import trained caffe model into dl4j?

**Pull request closed without merging**